app/src/manager/admin.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib import admin
+
+from .models import Block, Hotkey, Operator, Server, Subnet, SubnetSlot, ValidatorInstance
+
+
+# Admin Classes
+@admin.register(Subnet)
+class SubnetAdmin(admin.ModelAdmin):
+    list_display = ("name", "description", "get_operator_count")
+    search_fields = ("name",)
+    list_filter = ("operators",)
+
+    def get_operator_count(self, obj):
+        return obj.operators.count()
+
+    get_operator_count.short_description = "Number of Operators"
+
+
+@admin.register(SubnetSlot)
+class SubnetSlotAdmin(admin.ModelAdmin):
+    list_display = (
+        "subnet",
+        "blockchain",
+        "netuid",
+        "restart_threshold",
+        "reinstall_threshold",
+        "get_registration_block",
+        "get_deregistration_block",
+    )
+    search_fields = ("subnet__name", "netuid")
+    list_filter = ("blockchain",)
+
+    def get_registration_block(self, obj):
+        return obj.registration_block.serial_number if obj.registration_block else "N/A"
+
+    get_registration_block.short_description = "Registration Block"
+
+    def get_deregistration_block(self, obj):
+        return obj.deregistration_block.serial_number if obj.deregistration_block else "N/A"
+
+    get_deregistration_block.short_description = "Deregistration Block"
+
```

Comment:
add "registered" column which checks if we have a registered hotkey there which has not deregistered